lib/src/component_declaration/component_base.dart

```diff
@@ -275,23 +275,27 @@ typedef PropsModifier(Map props);
 abstract class UiProps
     extends Object with MapViewMixin, PropsMapViewMixin, ReactPropsMixin, UbiquitousDomPropsMixin, CssClassPropsMixin
     implements Map {
+
```

Comment:
#nit Extra whitespace. It's not convention to put whitespace at the beginning of a class/method body.